core/app/models/spree/shipment.rb

```diff
@@ -199,24 +199,6 @@ def selected_shipping_rate_id=(id)
       self.save!
     end
 
-    # Determines the appropriate +state+ according to the following logic:
-    #
-    # canceled   if order is canceled
-    # pending    unless order is complete and +order.payment_state+ is +paid+
-    # shipped    if already shipped (ie. does not change the state)
-    # ready      all other cases
-    def determine_state(order)
-      return 'canceled' if order.canceled?
-      return 'pending' unless order.can_ship?
-      return 'pending' if inventory_units.any? &:backordered?
-      return 'shipped' if state == 'shipped'
-      if order.paid? || !Spree::Config[:require_payment_to_ship]
```

Comment:
`Spree::Config[:require_payment_to_ship]` is set to `false` since we do not require payment to ship, which will cause this if statement to always be true and therefore always set the state of the other shipments (besides the one that had `#ready!` originally called on it) to have it's state set to ready.